README.md

```diff
@@ -1,23 +1,24 @@
-# Accurate Instagram Filters Reconstruction
+# Accurate Image Filters Reconstruction
```

Comment:
Image Filters could be differ: color, kernel, ML. As this is the main header, we should be more precise, the actual meaning is "Accurate Color Filters Reconstruction". But taking SEO in account we could be *more* precise and leave Instagram filters (even if we could capture other filters).

Reply:
K, that's good to go